osu.Game/Stores/BeatmapImporter.cs

```diff
@@ -315,21 +317,35 @@ public void Dispose()
         /// <summary>
         /// A dummy WorkingBeatmap for the purpose of retrieving a beatmap for star difficulty calculation.
         /// </summary>
-        private class DummyConversionBeatmap : WorkingBeatmap
+        private class DummyConversionBeatmap : IWorkingBeatmap
         {
-            private readonly IBeatmap beatmap;
+            public IBeatmap Beatmap { get; }
 
             public DummyConversionBeatmap(IBeatmap beatmap)
-                : base(beatmap.BeatmapInfo, null)
             {
-                this.beatmap = beatmap;
+                Beatmap = beatmap;
             }
 
-            protected override IBeatmap GetBeatmap() => beatmap;
-            protected override Texture? GetBackground() => null;
-            protected override Track? GetBeatmapTrack() => null;
-            protected internal override ISkin? GetSkin() => null;
-            public override Stream? GetStream(string storagePath) => null;
+            public IBeatmapInfo BeatmapInfo => throw new NotImplementedException();
+            public IBeatmapSetInfo BeatmapSetInfo => throw new NotImplementedException();
+            public IBeatmapMetadataInfo Metadata => throw new NotImplementedException();
+            public bool BeatmapLoaded => throw new NotImplementedException();
+            public bool BackgroundLoaded => throw new NotImplementedException();
+            public bool WaveformLoaded => throw new NotImplementedException();
+            public bool StoryboardLoaded => throw new NotImplementedException();
+            public bool SkinLoaded => throw new NotImplementedException();
+            public bool TrackLoaded => throw new NotImplementedException();
+            public Texture Background => throw new NotImplementedException();
+            public Waveform Waveform => throw new NotImplementedException();
+            public Storyboard Storyboard => throw new NotImplementedException();
+            public ISkin Skin => throw new NotImplementedException();
+            public Track Track => throw new NotImplementedException();
+            public IBeatmap GetPlayableBeatmap(IRulesetInfo ruleset, IReadOnlyList<Mod> mods, TimeSpan? timeout = null) => throw new NotImplementedException();
```

Comment:
Aside from the fact that this pull doesn't compile (which is resolved by merging `master`), this can't ever work. This dummy beatmap is used for diffcalc, and diffcalc needs `GetPlayableBeatmap()` to have an implementation. Not having one is causing `BeatmapImporterTests` to fail.